driver-core/src/main/com/mongodb/internal/client/model/FindOptions.java

```diff
@@ -52,6 +52,7 @@
     private Bson min;
     private boolean returnKey;
     private boolean showRecordId;
+    private boolean allowDiskUse;
```

Comment:
Since this isn't nullable, we can't actually do what the documentation suggests:

    This option is sent only if the caller explicitly provides a value. 

i.e. if the application explicitly sets it to false, the driver won't send it.

Should we make this nullable?

Reply:
Updated - we only send when true.

Reply:
I had a look at AggregateOperationImpl, and there the allowDiskUse field is defined as a Boolean rather than boolean.  The operation layer is not generally consistent with use of Boolean/boolean, but I'd prefer that we at least treat the same option similarly across different operations.

This will also make the driver in line with the spec, and not have to change again once we add command monitoring assertions that enforce which fields are actually sent to the server.